doc/api/process.md

```diff
@@ -1626,6 +1626,29 @@ if (process.getuid && process.setuid) {
 or Android).
 
 
+## process.shouldAbortOnUncaughtException
+<!-- YAML
+added: REPLACEME
+-->
+
+* {boolean}
+
+The `process.shouldAbortOnUncaughtException` switch controls the behavior
+when the `--abort-on-uncaught-exception` flag was passed on the command line
+or set through [`v8.setFlagsFromString()`][]:
+
+If the flag was passed to Node and `process.shouldAbortOnUncaughtException`
+is `true`, the process will abort when enountering an uncaught exception
+(regardless of any [`process.on('uncaughtException')`][] listeners).
+
+If the flag was passed to Node and `process.shouldAbortOnUncaughtException`
+is `false`, the process will not abort.
+
+If the `--abort-on-uncaught-exception` flag is not set, this value is ignored.
+
+*Note*: If your code uses the deprecated [`domain`][] built-in module,
+this flag will be set by that module on whenever domain state changes.
```

Comment:
I think there's an extra 'on' here. Could just be '[...]module whenever[...]'.

Reply:
done, thanks for catching!